src/CryptoAxus.Application/Features/Artist/GetOffersMadeByArtist/Request/GetOffersMadeByArtistRequestExample.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace CryptoAxus.Application.Features.Artist.GetOffersMadeByArtist.Request;
 
-public class GetOffersReceivedByArtistRequestExample : IExamplesProvider<GetOffersMadeByArtistRequest>
+public class GetOffersMadeByArtistRequestExample : IExamplesProvider<GetOffersMadeByArtistRequest>
```

Comment:
Why is this file changed?

Reply:
This was not changed on my end as, I have send the screenshot on slack